### DISCUSSION TITLE
Can't play with my deck online

### DISCUSSION THREAD

**Discussion by Trigga325** [BAA-Studios/MOOgiwara#75] [Q&A] [answered]
Hi, I build a deck but I can't seem to play online and it doesn't look like I'm using banned cards. Is there a reason I didn't consider? Also could you add a feature, that shows if cards can't be played in a deck?
Thanks for any help!

**Comment by KOOKIIEStudios** [comment, marked as answer]
NGL, your query got all of the staff members fairly confused, since we never got to the point of implementing the deck manager in MOOgiwara.
The current state of the project allows random matching of connected players, with preset dummy decks (for testing purposes), and playing legal moves - however, I don't believe you can actually win/lose a game yet, nor build a deck.

We're wondering if you stumbled upon this repository and mistook it for OPTCG Sim. This is a different project.

**Reply by Trigga325** [reply]
I'm sorry I confused this project with another. You can delete this question.
Sorry again and have a great one.
